Sources/core/objects/Human.cpp

```diff
@@ -385,6 +396,54 @@ void Human::CalculateVisible(std::list<point>* visible_list)
     }
 }
 
+void Human::Bump(id_ptr_on<IMovable> item)
+{
+    if(id_ptr_on<Projectile> p = item)
+    {
+        bool damaged = false;
+        if (p.valid())
+        {
+            health_ -= p->GetDamage();
+            if (id_ptr_on<Bullet> b = p)
+            {
+                 GetGame().GetChat().PostSimpleText(name + " got hit by a bullet!", owner->GetId());
+                 unsigned int value = GetRand() % 3;
+                 std::string snd;
+                 if (value == 0)
+                     snd = "genhit1.ogg";
+                 if (value == 1)
+                     snd = "genhit2.ogg";
+                 if (value == 2)
+                     snd = "genhit3.ogg";
+                 PlaySoundIfVisible(snd, owner.ret_id());
```

Comment:
`if` should be always with brackets (`{`, `}`). You probably have copied that code from another part of Griefly, but that means that another part should be fixed.